### ISSUE TITLE
Angular 2 - Create github project

### ISSUE THREAD

**Issue by danicomas** [isteven/angular-multi-select#382]
Angular 2 beta was released we should create or migrate this plugin to angular 2. @isteven

**Comment by isteven** [comment]
@danicomas , 

Yeah this is a priority now.. Thanks for reminding. Any good resource or example on how directive is defined in Angular 2?

**Comment by danicomas** [comment]
@isteven I did a component example in this blog https://www.ingens-networks.com/blog/post/2015/10/05/angular-2-components-creando-un-modal.aspx (Spanish) but you can find other examples like too: https://github.com/thelgevold/angular-2-samples/tree/master/components

**Comment by aciccarello** [comment]
I honestly have limited time but I'd be interested in being involved. Haven't done any Angular 2 development yet but I've been trying to pay close attention to the patterns.

**Comment by sudhishpathiyeri** [comment]
@isteven any work started on this?

**Comment by msrkrishnan** [comment]
@isteven - any update on this?

**Comment by ricetrac** [comment]
+1

**Comment by ferrarikarthik** [comment]
Any update on migrating this plugin to Angular 2 and above?

**Comment by pankajparkar** [comment]
@ferrarikarthik @ricetrac @msrkrishnan @danicomas I've created a multiselect plugin similar as that of isteven at least in terms of look and feel. Please have a look at it once.

https://github.com/ngx-lib/multiselect

**Comment by msrkrishnan** [comment]
@pankajparkar - good work, it looks good. Couple of suggestions:

1. If all items are selected, provide an option to show label as "All"
2. Also show number of items selected in label
3. Is it possible to customize Tick mark?

**Comment by pankajparkar** [comment]
@msrkrishnan thanks for your suggestion, I'll create an issue for each of the point and start working on it. Let me know if you're interested to contribution.

Followup question: I didn't get the 1st point, particularly *provide an option to show label as "All"*

**Comment by msrkrishnan** [comment]
If all the items in the drop down is selected, then we can just show "All Selected" or a custom label. It should be based on options/config. If user wants to see selected item names, leave as it is like Option, Option2, etc.,
